test/invoice_repository_test.rb

```diff
@@ -53,52 +85,72 @@ def test_random
   def test_find_all_by_id
     invoices = invoice_repo.find_all_by_id(1)
     invoices1 = invoice_repo.find_all_by_id(45)
+
     assert_equal 1, invoices.size
     assert_equal 0, invoices1.size
   end
 
   def test_find_all_by_merchant_id
     invoices = invoice_repo.find_all_by_merchant_id(1)
+
     assert_equal 3, invoices.size
   end
 
   def test_find_all_by_customer_id
     invoices = invoice_repo.find_all_by_customer_id(539)
+
     assert_equal 1, invoices.size
   end
 
   def test_find_all_by_status
     invoices = invoice_repo.find_all_by_status("shipped")
     invoices1 = invoice_repo.find_all_by_status("not shipped")
+    
     assert_equal 3, invoices.size
     assert_equal 0, invoices1.size
   end
 
   def test_find_all_by_created_at
-    invoices = invoice_repo.find_all_by_created_at(Date.parse("2012-03-27 14:54:09 UTC"))
-    invoices1 = invoice_repo.find_all_by_created_at(Date.parse("2016-01-01"))
+    found_date = Date.parse("2012-03-27 14:54:09 UTC")
+    not_found_date = Date.parse("2016-01-01")
+
+    invoices = invoice_repo.find_all_by_created_at(found_date)
+    invoices1 = invoice_repo.find_all_by_created_at(not_found_date)
+
     assert_equal 3, invoices.size
     assert_equal 0, invoices1.size
   end
 
   def test_find_all_by_updated_at
-    invoices = invoice_repo.find_all_by_updated_at(Date.parse("2012-03-27 14:54:09 UTC"))
-    invoices1 = invoice_repo.find_all_by_updated_at(Date.parse("2016-01-01"))
+    found_date = Date.parse("2012-03-27 14:54:09 UTC")
+    not_found_date = Date.parse("2016-01-01")
+
+    invoices = invoice_repo.find_all_by_updated_at(found_date)
+    invoices1 = invoice_repo.find_all_by_updated_at(not_found_date)
+
     assert_equal 3, invoices.size
     assert_equal 0, invoices1.size
   end
 
   def test_create_standardizes_input_for_invoice
-    input = {customer: "customer", merchant: "merchant", status: "shipped",
-    items: ["item1", "item2", "item3"]}
+    input = {
+      customer: "customer",
+      merchant: "merchant",
+      status: "shipped",
+      items: ["item1", "item2", "item3"]
+    }
+
     sales_engine.expect(:create_invoice_items_from_inputs_and_invoice, nil, [4, input])
-    # output = invoice_repo.create(input)
-  #  assert_equal "customer", output.customer_id
   end
 
   def test_it_delegates_create_invoice_items_to_sales_engine
-    input = {customer: "customer", merchant: "merchant", status: "shipped",
-      items: ["item1", "item2", "item3"]}
+    input = {
+      customer: "customer",
+      merchant: "merchant",
+      status: "shipped",
+      items: ["item1", "item2", "item3"]
+      }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.